eZ/Publish/API/Repository/Tests/TrashServiceTest.php

```diff
@@ -684,6 +685,51 @@ public function testFindTrashItems()
         $this->assertEquals(4, $searchResult->totalCount);
     }
 
+    /**
+     * @covers  \eZ\Publish\API\Repository\TrashService::findTrashItems
+     *
+     * @depends eZ\Publish\API\Repository\Tests\TrashServiceTest::testTrash
+     *
+     * @throws \ErrorException
+     * @throws \eZ\Publish\API\Repository\Exceptions\ForbiddenException
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException
+     */
+    public function testFindTrashItemsSortedByDateTrashed(): void
+    {
+        $repository = $this->getRepository();
+        $trashService = $repository->getTrashService();
+        $locationService = $repository->getLocationService();
+
+        $folder1 = $this->createFolder(['eng-GB' => 'Folder1'], 2);
+        $folder2 = $this->createFolder(['eng-GB' => 'Folder2'], 2);
+
+        $firstTrashedItem = $trashService->trash(
+            $locationService->loadLocation($folder1->contentInfo->mainLocationId)
+        );
+        $this->updateTrashedDate($firstTrashedItem->id, \time() - 100);
+        $latestTrashItem = $trashService->trash(
+            $locationService->loadLocation($folder2->contentInfo->mainLocationId)
+        );
+
+        $query = new Query();
+        $query->filter = new Criterion\LogicalAnd([]);
```

Comment:
Please narrow it down instead of using this artificial Criterion. So e.g. `Criterion\ContentId([$folder1->id, $folder2->id])` or sth like that.